src/reaper_osara.cpp

```diff
@@ -4025,6 +4025,61 @@ void cmdMoveItemsOrEnvPoint(Command* command) {
 	}
 }
 
+void cmdInsertOrMoveSpecificMarker(Command* command) {
+	int cmd = command->gaccel.accel.cmd;
+	int beforeCount = CountProjectMarkers(0, nullptr, nullptr);
+	int wantNum;
+	// Work out the desired marker based on the command ID.
+	if (cmd == 40656)
+		wantNum = 10;
```

Comment:
I'd really prefer blocks to be braced, even if they're only a single statement, but it's not a big deal.